packages/backend/init-action/helpers/history.js

```diff
@@ -1,24 +1,21 @@
-const core = require('@actions/core');
```

Comment:
Need to get our eslints in order... was this because you weren't inside the folder when you created it?